buildSrc/gradle.lockfile

```diff
@@ -1,33 +1,28 @@
 # This is a Gradle generated file for dependency locking.
 # Manual edits can break the build and are not advised.
 # This file is expected to be part of source control.
-com.googlecode.javaewah:JavaEWAH:1.1.13=buildScriptClasspath,compileClasspath,testCompileClasspath,testRuntimeClasspath
```

Comment:
this file needed update, for some reasone we had some of version 1.7.* while others with 1.9.*